include/graphics.h

```diff
@@ -4778,91 +4778,41 @@ extern const u32 gPartyMenuBg_Pal[];
 
 // berry pics
 extern const u32 gBerryPic_Cheri[];
-extern const u32 gBerryPalette_Cheri[];
```

Comment:
Initial comment while I'm in public transit:
I understand reordering to match the constant order, but I what I don't get is why change the Pic, Palette, Pic, Pallete order that pret has:
https://github.com/pret/pokeemerald/blob/6e5520207f91ce883ee0cf86a4f8552fb7a5658c/include/graphics.h#L3921

Reply:
Declaration order doesn't matter when it comes to how things are linked, especially since these are just externs. It doesn't affect matching so the order was done in whatever it was originally dumped as for pret's repo.

What I've done is match the organization for what the newly-added Berry pics were like (Occa, etc.) and I also noticed the definitions (in `src/data/graphics/berries.h`) were in this order so that's more reason to have this order.